deploy/storage/base_bigtable_info.yaml

```diff
@@ -1,14 +1,14 @@
 project: datcom-store
 instance: prophet-cache
 tables:
-  - auto1d_2023_10_30_02_15_26
-  - auto1w_2023_10_29_21_01_13
-  - auto2w_2023_11_14_09_32_56
-  - biomedical_2023_11_02_03_52_24
-  - country_2023_10_30_10_25_19
-  - disaster_2023_10_29_21_30_29
-  - frequent_2023_10_30_05_23_27
-  - infrequent_2023_11_13_16_43_32
-  - ipcc_2023_11_01_03_46_35
-  - place_2023_10_26_12_52_26
-  - schema_2023_10_30_00_25_48
+  - auto1d_2024_01_03_02_24_25
```

Comment:
Based on https://screenshot.googleplex.com/44vmpJcnuXv3nKR, suspicious import groups (@n-h-diaz):

* auto1d_2024_01_03_02_24_25
* auto1w_2023_12_31_21_01_04
* auto2w_2023_12_26_09_34_57
* disaster_2023_12_31_21_32_35

Reply:
could you please try updating with the following builds (which should hopefully avoid the buggy place builds)

* auto1d_2024_01_04_02_24_07
* auto1w_2023_12_17_21_04_54
* auto2w_2023_12_12_09_41_21
* disaster_2023_12_17_21_37_47

Reply:
Done.